### PR TITLE
ADD SCORES AND GAME FINISHED:

### PR DESCRIPTION
**Pull request by riblanc**
Add basic logic when game is finished, and set scores on frontend

### CODE REVIEW

File: back/src/game/gameplay.service.ts [hunk 1]
```diff
@@ -301,7 +305,38 @@ export class GameplayService {
 			}, 1000 / frameRate);
 		} else {
 			// TODO: insert game history in database
```
Comment (AdrianWeaver) [inline back/src/game/gameplay.service.ts:307]:
```suggestion
```

File: back/src/game/gameplay.service.ts [hunk 2]
```diff
@@ -40,7 +42,7 @@ export class GameplayService {
 		this.player_2.socket?.emit('game:start', { side: 'right' });
 
 		this.logger.log(`Game started between ${this.player_1.socket.id} and ${this.player_2.socket.id}`);
-		this.startDebugLogger();
+		//this.startDebugLogger();
```
Comment (AdrianWeaver) [inline back/src/game/gameplay.service.ts:45]:
```suggestion
```

File: back/src/game/gameplay.service.ts [hunk 3]
```diff
@@ -301,7 +305,38 @@ export class GameplayService {
...
+				include: { records: { include: { player: true } } }
+			});
+
+			this.player_1.socket.emit('game:finished', {
+				win: this.player_1.userId === this.winner.id
+			});
+			this.player_2.socket.emit('game:finished', {
+				win: this.player_2.userId === this.winner.id
+			});
+			//this.emitToPlayers('game:finished', )
```
Comment (AdrianWeaver) [inline back/src/game/gameplay.service.ts:338]:
```suggestion
```

**Review by AdrianWeaver — APPROVED** [review]
Nice ! We're getting closer!

**Review by riblanc — COMMENTED** [review]
noice

**Review by Deacllock — APPROVED** [review]

**Pull request merged**